src/librustc_codegen_ironox/basic_block.rs

```diff
@@ -6,5 +6,48 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+use context::CodegenCx;
+use value::Value;
+
+macro_rules! asm {
```

Comment:
You're right, I'll change that.

(This will eventually be a method of the `IronOxModule`. I'll change the type of `instructions` in `BasicBlockData` to `Vec<Instruction>`. Calling `asm` on the module will return a `String` of x86-64 instructions.)